src/main/java/net/dv8tion/jda/api/requests/restaction/AbstractThreadCreateAction.java

```diff
@@ -36,7 +35,7 @@
  * @param <R>
  *        The common return type of setters, allowing for fluid interface design
  */
-public interface AbstractThreadCreateAction<T, R extends AbstractThreadCreateAction<T, R>> extends FluentRestAction<T, R>
+public interface AbstractThreadCreateAction<T, R extends AbstractThreadCreateAction<T, R>> extends RestAction<T>
```

Comment:
Is it not a problem that this is no longer Fluent? not even overriding?

Reply:
Not really an issue no